src/components/views/rooms/MessageComposer.tsx

```diff
@@ -219,6 +222,7 @@ interface IState {
 @replaceableComponent("views.rooms.MessageComposer")
 export default class MessageComposer extends React.Component<IProps, IState> {
     private dispatcherRef: string;
+    // @ts-ignore
     private messageComposerInput: SendMessageComposer;
```

Comment:
This uses an old way of collecting the reference in React.
Can we modify it to using a `RefObject` instead? And with that, we should be able to get rid of the `@ts-ignore` above